### ISSUE TITLE
cargo suply manifests do not always have the correct ship name in the destination section

### ISSUE THREAD

**Issue by John-Gregson** [BeeStation/NSV13#2001]
## Reproduction: ordered 6 crates on the atlas (2 pdc ammo crates 2 nac shell crates and 2 gunpowder crates) when stamping the supply manifests i realised that 2 of the manifests had different names to the others. the ships name at the time was "NSV Weed Smoker" and 4 of the manifests had the correct name of "NSV Weed Smoker" while the 2 others had the names "NSV Thor" and "NSV Corvus". for reproduction I suppose order 6 crates on the atlas and check what it says under destination 

screenshots below 

![Screenshot (1896)](https://user-images.githubusercontent.com/97245342/180661140-ec7cdc0f-18d8-4360-adab-f3995623254b.png)
![Screenshot (1898)](https://user-images.githubusercontent.com/97245342/180661390-e0d5958e-b205-465d-898d-da993223d335.png)

**Comment by Bokkiewokkie** [comment]
Yep, the game calls the proc to create a *new* station name, instead of actually looking for the actual name.
https://github.com/BeeStation/NSV13/blob/da9faec150a40dda84b22eddee9bb563f31d0c02/code/modules/cargo/order.dm#L46

**Comment by Bobbanz1** [comment]
Issue has been fixed as of the Beebase @Bokkiewokkie
